python/metatomic_torch/tests/symmetrized_ase_calculator.py

```diff
@@ -0,0 +1,396 @@
+import numpy as np
+import pytest
+from ase import Atoms
+from ase.build import bulk, molecule
+
+from metatomic.torch.ase_calculator import SymmetrizedCalculator, _get_quadrature
+
+
+def _body_axis_from_atoms(atoms: Atoms) -> np.ndarray:
+    """
+    Return the normalized vector connecting the two farthest atoms.
+
+    :param atoms: Atomic configuration.
+    :return: Normalized 3D vector defining the body axis.
+    """
+    pos = atoms.get_positions()
+    if len(pos) < 2:
+        return np.array([0.0, 0.0, 1.0])
+    d2 = np.sum((pos[:, None, :] - pos[None, :, :]) ** 2, axis=-1)
+    i, j = np.unravel_index(np.argmax(d2), d2.shape)
+    b = pos[j] - pos[i]
+    nrm = np.linalg.norm(b)
+    return b / nrm if nrm > 0 else np.array([0.0, 0.0, 1.0])
+
+
+def _legendre_0_1_2_3(c: float) -> tuple[float, float, float, float]:
+    """
+    Compute Legendre polynomials P0..P3(c).
+
+    :param c: Cosine between the body axis and the lab z-axis.
+    :return: Tuple (P0, P1, P2, P3).
+    """
+    P0 = 1.0
+    P1 = c
+    P2 = 0.5 * (3 * c * c - 1.0)
+    P3 = 0.5 * (5 * c * c * c - 3 * c)
+    return P0, P1, P2, P3
+
+
+class MockAnisoCalculator:
```

Comment:
Instead of making this an ase Calculator, could we make it a metatomic model? This was we can re-use it if/when we move to make rotational average available for everyone.